
Submitted By:            Daniel Schreij <contact at dschreij at gmail dot com>
Date:                    2016-11-16
Initial Package Version: 2.0.0
Upstream Status:         Reported
Origin:                  https://github.com/nigels-com/glew/blob/7d72257b0ff2aadab92d9b627beb44c36b072e52/build/cmake/CMakeLists.txt
Description:             Fixes memset not found error with VS2008 by disabling -nodefaultlib command.

--- build/cmake/CMakeLists.txt  2016-11-16 22:05:02.000000000 -0400
+++ build/cmake/CMakeLists.txt  2016-11-16 22:10:10.000000000 -0400
@@ -101,5 +101,5 @@
   target_compile_options (glew_s PRIVATE -GS-)
   # remove stdlib dependency
-  target_link_libraries (glew LINK_PRIVATE -nodefaultlib -noentry)
+  # target_link_libraries (glew LINK_PRIVATE -nodefaultlib -noentry)
 elseif (WIN32 AND ((CMAKE_C_COMPILER_ID MATCHES "GNU") OR (CMAKE_C_COMPILER_ID MATCHES "Clang")))
   # remove stdlib dependency on windows with GCC and Clang (for similar reasons